src/nvim/search.c

```diff
@@ -1353,6 +1353,9 @@ int do_search(oparg_T *oap, int dirc, int search_delim, char_u *pat, long count,
     }
     retval = 1;                     // pattern found
 
+    if (sia && sia->sa_wrapped)
+      apply_autocmds(EVENT_SEARCHWRAPPED, NULL, NULL, false, NULL);
```

Comment:
Per the style guide, please use curly braces even for single line if statements.

```suggestion
    if (sia && sia->sa_wrapped) {
      apply_autocmds(EVENT_SEARCHWRAPPED, NULL, NULL, false, NULL);
    }
```